src/index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width">
     <title>Esri Global Nav Demo Pages</title>
     <link rel="stylesheet" href="./esri-global-nav.css">
-    <link rel="stylesheet" href="https://s3-us-west-1.amazonaws.com/patterns.esri.com/files/calcite-web/1.0.1/css/calcite-web.min.css">
+    <link rel="stylesheet" href="https://s3-us-west-1.amazonaws.com/patterns.esri.com/files/calcite-web/1.2.5/css/calcite-web.min.css">
     <link rel="stylesheet" href="./syntax.css">
```

Comment:
I know this is just the demo page but I always like to use https://webapps-cdn.esri.com/CDN/jslibs/calcite/1.25/css/calcite-web.min.css as it's AKAMI cached

Reply:
nice, good catch.